### DISCUSSION TITLE
help pls

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#7967] [QA - tech support + general]
![Screen Shot 2021-09-30 at 2 35 01 PM](https://user-images.githubusercontent.com/82217573/135519423-225047e2-9a06-4877-9b28-8c43b19edc8a.png)

**Comment by multimokia** [comment]
Do you have submods installed at all? I'm seeing a bunch of stuff which shouldn't be there

**Comment by ghost** [comment]
i fixed it just now. the problem was that I put the rpy files n the game folder AND the submod folder

**Comment by ghost** [comment]
btw i cant get the colorful ui for it. how do I get it?

**Comment by multimokia** [comment]
Not sure, along with not being sure what submod you installed. You'll want to see if you can find more info
